cloudflare/data_source_zones_test.go

```diff
@@ -2,14 +2,45 @@ package cloudflare
 
 import (
 	"fmt"
+	"log"
+	"os"
 	"strconv"
 	"testing"
 
+	"github.com/cloudflare/cloudflare-go"
 	"github.com/hashicorp/terraform/terraform"
 
 	"github.com/hashicorp/terraform/helper/resource"
 )
 
+func init() {
+	resource.AddTestSweepers("cloudflare_zones", &resource.Sweeper{
+		Name: "cloudflare_zones",
+		F:    testSweepCloudflareZones,
+	})
+}
+
+func testSweepCloudflareZones(r string) error {
+	client, _ := cloudflare.New(os.Getenv("CLOUDFLARE_TOKEN"), os.Getenv("CLOUDFLARE_EMAIL"))
+	zones, _ := client.ListZones("baa.com", "baa.net", "baa.org", "foo.net")
```

Comment:
Could you please add error check here?

Reply:
Good pick up! I've included this as a part of 7878e60.